content/App.tsx

```diff
@@ -115,17 +118,32 @@ const App: FC<AppProps> = (props) => {
           }}
         />
       </Head>
-      <ThemeProvider theme={theme}>
-        <RecoilRoot>
-          <QueryClientProvider client={queryClient} contextSharing={true}>
-            <WithAppContainers>
-              <MediaContextProvider>
-                <InnerApp {...props} />
-              </MediaContextProvider>
-            </WithAppContainers>
-          </QueryClientProvider>
-        </RecoilRoot>
-      </ThemeProvider>
+      <ChakraProvider
+        theme={extendTheme({
+          ...chakraTheme,
+          // TODO: Global V2 Styles, need replacing
```

Comment:
Should we do this todo before merging?

Reply:
I think it makes sense to remove this once we have removed the themeprovider that comes with styled-components. Chakra is just overwriting some styles on the body currently.